File: app/src/main/java/com/zyj/dribbbleclient/app/api/DribbbleService.java
package com.zyj.dribbbleclient.app.api;

import com.zyj.dribbbleclient.app.model.Shot;

import retrofit.http.GET;
import rx.Observable;

public interface DribbbleService {

    @GET("shots/")
    Observable<Shot[]> shotList();
}

File: app/src/main/kotlin/com/zyj/dribbbleclient/app/api/DribbbleService.kt
package com.zyj.dribbbleclient.app.api

import com.zyj.dribbbleclient.app.model.Shot

import retrofit.http.GET
import retrofit.http.Query
import rx.Observable

interface DribbbleService {

    @GET("shots/")
    fun shotList(@Query("list") list: String? = null,
                 @Query("sort") sort: String? = null): Observable<Array<Shot>>
}
